ocs_ci/ocs/rados_utils.py

```diff
@@ -273,6 +277,76 @@ def get_mgr_proxy_container(self, node, docker_image, proxy_container="mgr_proxy
 
         return mgr_object
 
+    def verify_cephblockpool_status(
+        self,
+        pool_name=constants.DEFAULT_BLOCKPOOL,
+        namespace=None,
+        required_phase=constants.STATUS_READY,
+    ):
+        """
+        Verify the phase of cephblockpool
+
+        Args:
+            pool_name (str): The name of the Ceph block pool
+
+        Returns:
+            status: True if the Ceph block pool is in Ready status, False otherwise
+        """
+        if not namespace:
+            namespace = config.ENV_DATA["cluster_namespace"]
+        cmd = (
+            f"oc get {constants.CEPHBLOCKPOOL} {pool_name} -n {namespace} "
+            "-o=jsonpath='{.status.phase}'"
+        )
+        # phase = run_cmd(cmd=cmd)
+
+        phase = retry((CommandFailed), tries=20, delay=10,)(
+            run_cmd
+        )(cmd=cmd)
+
+        logger.info(f"{pool_name} is in {phase} phase")
+        logger.info(f"Required phase is {required_phase}")
+        if phase == required_phase:
```

Comment:
instead of if-else use the line bellow. it is cleaner and more pythonic 

`return phase == required_phase`

Reply:
Addressed in latest commit.